recipes/msprime/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "msprime" %}
+{% set version = "0.5.0b2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "5cf78da4c27669d3980d86861cbcdb67ee3ead56d94206e52bc7e0b208b276d9" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - mspms=msprime.cli:mspms_main
+    - msp=msprime.cli:msp_main
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win and py<35]
+
+# Numpy version requirement copied from conda-forge h5py feedstock
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+    - svgwrite
+    - six
+    - numpy 1.8.*   # [not (win and (py35 or py36))]
+    - numpy 1.9.*   # [win and py35]
+    - numpy 1.11.*  # [win and py36]
+    - hdf5
+    - gsl 
+  run:
+    - python
+    - svgwrite
+    - six
+    - numpy >=1.8   # [not (win and (py35 or py36))]
+    - numpy >=1.9   # [win and py35]
+    - numpy >=1.11  # [win and py36]
+    - hdf5
```

Comment:
Please make sure to check that users are including pinnings for `hdf5`. Hopefully we won't have to do this too much longer once `conda-build` 3 is out.